doc/ref_cert/RC2/chapters/chapter04.md

```diff
@@ -3,3 +3,38 @@
 # 4. CNF Test Cases and Requirements Traceability
 
 ## Table of Contents
+* [4.1 Introduction](#4.1)
+* [4.2 Selection Criteria](#4.2)
+* [4.3 Traceability Matrix](#4.3)
+  * [4.3.1Test Case Traceability to RA2 Requirements](#4.3.1)
+  
+<a name="4.1"></a>
+## 4.1 Introduction
+The scope of this chapter is to identify and list down test cases based on requirements defined in [Reference Architecture-2 (RA-2)](../../../ref_arch/kubernetes/README.md). This will serve as traceability between test cases and requirements for CNF interoperability.
+
+Note that each requirement may have one or more test cases associated with it.
+
+<a name="4.2"></a>
+## 4.2 Selection Criteria
+Test cases and their dependencies must be open source. The test cases (or test suite with the test case) as well as the environment needed to run the test should be reproducible by any party following publicly available documentation.
+
+Examples of initiatives (having testing tools, test suites, etc) with test cases which could be used include K8s Conformance, K8s e2e, Sonobuoy, Anuket FuncTest, CNF Conformance.
+
+<a name="4.3"></a>
+## 4.3 Traceability Matrix
+
+The following is a Requirements Traceability Matrix (RTM) mapping Test Case, and/or Test Case Coverage, to RM and RA requirements -- configuration, deployment, runtime.
+
+<a name="4.3.1"></a>
+### 4.3.1 Test Case Traceability to RA2 Requirements
+This section focuses on the test case covering requirements in [RA-2 Chapter 4.9](https://github.com/cntt-n/CNTT/blob/master/doc/ref_arch/kubernetes/chapters/chapter04.md#49-kubernetes-workloads) for Kubernetes workloads.
+
+| RM/RA Ref | High-level test definition | Test name and project | Priority |
+|---|---|---|---|
+| ra2.app.001 | | | Must |
```

Comment:
I don't see the point to add blank tabs.

Reply:
They will be filled in later

Reply:
the requirements are already in RA2